src/modules/RA/ra-modules.js

```diff
@@ -150,6 +151,11 @@ export const config = {
   ],
 };
 
+if (process.env.REACT_APP_NOT_DISPLAY_BASELAYERS) {
+  const { menu } = config;
+  config.menu = menu.filter(obj => obj.label !== 'baseLayer.project');
+}
+
```

Comment:
Maybe you could set a more generic stuff by setting a "pseudo array".
For example in the .env file you've got the following line ⬇️ 
`REACT_APP_NOT_DISPLAY_MODULES="baseLayer foo bar"`

And then in the code you can iterate on it
```js
if (process.env.REACT_APP_NOT_DISPLAY_MODULES) {
  const { menu } = config;
  process.env.REACT_APP_NOT_DISPLAY_MODULES.split(' ').forEach(module => {
   config.menu = menu.filter(({ label }) => label !== module);
  }
}
```

Reply:
ok i miss `.project` but it's possible with a .map after the split.
What do you think?

Reply:
I like this idea

Reply:
I applied the changes
There is now a env variable looking like REACT_APP_NOT_DISPLAY_MODULES="baseLayer.project foo.project bar.project"

Reply:
Yes top!
Maybe you can avoid the repetition of '.project' in the env file and match the correspondance here
```js
const notDisplayModulesProject = !process.env.REACT_APP_NOT_DISPLAY_MODULES.split(' ').map(module => `${module}.project`}
config.menu = menu.filter(({ label }) => notDisplayModulesProject.includes(label));`
```

or but maybe less accurate
```js
config.menu = menu.filter(({ label }) => !process.env.REACT_APP_NOT_DISPLAY_MODULES.split(' ').includes(label.replace('.project', ''));
```